addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -69,149 +64,136 @@
+            <div class="o_we_bg_filter bg-black-50"/>
+            <h3 class="display-4">Innovation <br class="d-none d-lg-inline"/>Hub</h3>
+            <p class="lead">Where ideas come to life</p>
         </div>
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 7 / 11 / 13; z-index: 2;">
-            <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item justify-content-start o_colored_level shadow rounded-3 overflow-hidden o_cc o_cc3 o_draggable oe_img_bg o_bg_img_center g-height-9 g-col-lg-6 col-lg-6" data-name="Block" style="z-index: 2; grid-area: 1 / 1 / 10 / 7; background-image: url(/web/image/website.s_masonry_block_default_image_2); --grid-item-padding-y: 30px; --grid-item-padding-x: 30px;">
+            <div class="o_we_bg_filter bg-black-50"/>
+            <h3 class="display-4">Key <br class="d-none d-lg-inline"/>Milestone</h3>
+            <p class="lead">Reaching new heights together</p>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_image_texts_image_template" groups="base.group_user">
     <div class="row o_grid_mode" data-row-count="10">
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 1 / 11 / 4; z-index: 1;">
-            <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item g-height-10 o_grid_item_image text-center g-col-lg-3 col-lg-3" data-name="Block" style="grid-area: 1 / 9 / 11 / 12; z-index: 4;">
+            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 4 / 6 / 7; z-index: 2;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item g-height-5 text-center o_cc o_cc4 g-col-lg-4 col-lg-4" data-name="Block" style="z-index: 3; grid-area: 6 / 5 / 11 / 9;">
+            <h3>Community <br class="d-none d-lg-inline"/>Focus</h3>
+            <p>Building connections</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 6 / 4 / 11 / 7; z-index: 3;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item text-center o_cc o_cc4 g-col-lg-1 g-height-5 col-lg-1 o_snippet_mobile_invisible" data-name="Block" style="z-index: 3; grid-area: 6 / 12 / 11 / 13;">
```

Comment:
This block is not necessary on mobile, we could hide it for small breakpoint.
```suggestion
        <div class="o_grid_item text-center o_cc o_cc4 g-col-lg-1 g-height-5 d-lg-flex d-none o_snippet_mobile_invisible col-lg-1 o_snippet_mobile_invisible" data-name="Block" style="z-index: 3; grid-area: 6 / 12 / 11 / 13;">
```

Same for the other at line 96